### PR TITLE
updates for timer on stored_request

### PR DESCRIPTION
**Pull request by GLStephen**

### CODE REVIEW

File: endpoints/openrtb2/auction.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ import (
 	"github.com/prebid/prebid-server/version"
 )
 
-const storedRequestTimeoutMillis = 50
+const storedRequestTimeoutMillis = 500
```
Comment (SyntaxNode) [inline endpoints/openrtb2/auction.go:55]: Changing this time might have negative consequences for some hosts. I recommend either making this change in a fork or altering this PR to make it a configuration. I vaguely remember a PR a while back to make it configurable where I think the author lost interest. Happy if you'd like to pick that back up.

Comment (bsardo) [reply endpoints/openrtb2/auction.go:55]: Hi @GLStephen, just curious what your thoughts are here. Thanks!

**Comment by GLStephen** [comment]
@bsardo @SyntaxNode I don't think I intended to submit this as an upstream PR, sorry. This was a change we made in our fork.

**Comment by GLStephen** [comment]
PR submission was unintended

**Pull request closed without merging**